composer.json

```diff
@@ -27,6 +27,13 @@
 	"autoload": {
 		"classmap": ["app/Booting.php"]
 	},
+	"autoload-dev": {
```

Comment:
I'd add it under `autoload`?

Reply:
It is option. But really does (have) phpstan on production?